pkg/config/setup/config.go

```diff
@@ -646,6 +646,7 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("jmx_statsd_client_buffer_size", 0)
 	// the socket timeout (SO_SNDTIMEO) is in milliseconds
 	config.BindEnvAndSetDefault("jmx_statsd_client_socket_timeout", 0)
+	config.BindEnvAndSetDefault("jmx_fips", false)
```

Comment:
Non-blocking nit: For consistency with other config flags, it may be good to use `jmx_fips.enabled` or `jmx.fips.enabled`. I do see that JMX config options tend to be rather unique with their use of unseparated snake case config names so not a big deal either way.